ob_v2p1/ob-spec-v2p1.md

```diff
@@ -0,0 +1,540 @@
+
+## Properties
+Below are listed several properties usable across several Classes. They are optional in all cases.
+
+### related
+The <code>related</code> property identifies another entity of the same type that should be considered the same for most purposes. It is primarily intended to identify alternate language editions or previous versions of BadgeClasses. See examples: <a data-cite="OB-EXAMPLES#Internationalization">alternate language versions</a> and <a data-cite="OB-EXAMPLES#badgeclass-version-control">BadgeClass version control</a>.
+
+### version
+The <code>version</code> property allows issuers to specify a version string or number. It is primarily used to update a BadgeClass without changing the meaning of previously awarded Assertions by duplicating and linking to the previous version. See <a data-cite="OB-EXAMPLES#badgeclass-version-control">example</a>.
+
+Property | Expected Type | Description
+---------|---------------|-----------
+related  | @id ([Multiple values allowed](#array)) | Identifies a related version of the entity.
+version  | Text or Number | The version identifier for the present edition of the entity.
+endorsement | @id: [Endorsement](#Endorsement) Multiple values allowed) | A claim made about this entity. Note: As endorsements must be published after the publication of the entity they endorse, it will not always be possible to establish a two-way linkage with this property.
+
+
+<h2 id="ProfileIdentifierProperties">Profile Identifier Properties</h2>
+
+When profiles are referenced elsewhere in the Open Badges Specification, they may be identified precisely by dereferencable id, such as when a BadgeClass links to an issuer Profile by its <code>id</code> URL. Other times, such as when identifying the [recipient](#recipient) of an [Assertion](#Assertion), Profiles may be identified by the value of a specific property unique to the individual or organization represented in a Profile. All properties that serve as profile identifiers must have values with a string datatype.
+
+**Properties considered serviceable identifiers include:**
+
+* <code>email</code>
+* <code>url</code>
+* <code>telephone</code>
+
+Many platforms that allow badge issuers and recipients to establish their identities as Profiles support only <code>email</code> (<code>https://schema.org/email</code>) as an identifier property.
+
+
+<h2 id="Extensions">Extensions</h2>
+
+The 1.1 version of the Open Badges Specification introduces Extensions as a means for issuers to add additional metadata to Badge Objects beyond what the standard specifies itself. Additional properties are allowed without using Extensions, but Extensions allow issuers to declare how they are adding information so that it can be understood by others and other issuers can add the same sort of information in a compatible way.
+
+See the [[[OB-EXTENSIONS]]] page for specific examples and extensions ready to use in Badge Objects.
```

Comment:
The current spec references https://www.imsglobal.org/sites/default/files/Badges/OBv2p0Final/extensions/index.html for this link, and this proposed change links to what appears to be a page in the IMS CMS, and not under version control.